Testing/na/na_test.c

```diff
@@ -161,6 +161,9 @@ na_test_parse_options(int argc, char *argv[], struct na_test_info *na_test_info)
                 na_test_info->max_contexts =
                     (na_uint8_t) atoi(na_test_opt_arg_g);
                 break;
+            case 'Z': /* msg size */
+                na_test_info->max_msg_size = atoi(na_test_opt_arg_g);
+                break;
```

Comment:
@gnuoyd also added a -Z option to run the NA tests. Just thinking that you may want to use it with UCX too.